### ISSUE TITLE
Cannot send datetime to POST /projects

### ISSUE THREAD

**Issue by gideonthomas** [mozilla/publish.webmaker.org#93]
```js
{
        title: projectName,
        user_id: req.session.publishUser.id,
        date_created: '2015-06-03T13:21:58+00:00',
        date_updated: '2015-06-03T13:21:58+00:00'
}
```

Is what I'm sending

**Comment by gideonthomas** [comment]
I see a log:
```
insert into "projects" ("date_created", "date_updated", "description", "tags", "title", "user_id") values ($1, $2, $3, $4, $5, $6) returning "id" - date/time field value out of range: "1433337718000"
```

**Comment by ashleygwilliams** [comment]
are you sending that to `POST /projects`

**Comment by ashleygwilliams** [comment]
i've reproduced this locally, working on it now.

**Comment by ashleygwilliams** [comment]
the error seems to have to do with timezone

```
$ curl -X POST --data "title=X&user_id=1&date_created='2015-06-19T01:01:56+00:00&date_updated='2015-06-19T01:01:56+00:00'" http://localhost:2015/projects

[2015-06-19T19:56:35.151Z] ERROR: publish.webmaker.org/88022 on Ashleys-MacBook-Pro.local: insert into "projects" ("date_created", "date_updated", "description", "tags", "title", "user_id") values ($1, $2, $3, $4, $5, $6) returning "id" - invalid input syntax for type timestamp with time zone: "NaN"
```

**Comment by ashleygwilliams** [comment]
```
curl -X POST --data "title=X&user_id=1&date_created='Thu Jun 18 2015 21:01:56 GMT-0400 (EDT)'&date_updated='Thu Jun 18 2015 21:01:56 GMT-0400 (EDT)'" http://localhost:2015/projects

[2015-06-19T19:59:29.768Z] ERROR: publish.webmaker.org/88022 on Ashleys-MacBook-Pro.local: insert into "projects" ("date_created", "date_updated", "description", "tags", "title", "user_id") values ($1, $2, $3, $4, $5, $6) returning "id" - date/time field value out of range: "1434675716000"
```

**Comment by ashleygwilliams** [comment]
fixes in 2a7b490daadb1aad703bdb6fad09a80e0eb03592, bf9333759f52f94650dfc1e31f84b5517290aa89
